GoogleForCreators/web-stories-wp
# Shapes: Simplified circle path

## Context

This would change every story with a circle in it, making it slightly larger than it was when originally designed. Not sure this is undesirable.

We can merge this as-is or do an alternative:
1. ~Merge as-is impacting every story and template with a circle mask~
2. ~Make a migration slightly reducing the size of all circle masks already used in a story (would probably be imprecise, but could come close)~
3. Keep the old circle as it was, retire the mask, and a new circle (`CIRCLE2`) with the proper dimensions, only show the new circle in the mask library. All old stories would remain unchanged and all new stories would only use the new "proper" circle.

## Todo

- [x] Resolve migration issue

## User-facing changes

| Before | After |
|-|-|
| <img width="448" alt="Screen Shot 2022-12-20 at 23 26 43" src="https://user-images.githubusercontent.com/637548/208779583-046ff534-3f65-434f-be64-0d80535a2d7e.png"> | <img width="448" alt="Screen Shot 2022-12-20 at 23 22 20" src="https://user-images.githubusercontent.com/637548/208779608-e012d1ad-7089-4d7e-974c-21de3616abf9.png"> |

## Testing Instructions

This PR can be tested by following these steps:

1. Add a circle to the story from the mask library
2. Observe that the circle fills up the maximum amount of space inside the frame of the shape

## Checklist




---

Fixes #11579
